resources/devices/mobile-phone/nokia.json

```diff
@@ -51,6 +51,19 @@
             "isTablet": "false"
```

Comment:
A lot of new devices here, but I also tried to make sure any devices I tested matched the actual device (proper common name, touchscreen or not, etc...).